Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js

```diff
@@ -974,13 +974,25 @@ Ext.onReady(function () {
                 devids,
                 me = this,
                 isOrganizer = true;
+            
+            var rootOrganizers = [
+                "/zport/dmd/Devices",
+                "/zport/dmd/Locations",
+                "/zport/dmd/Groups",
+                "/zport/dmd/Systems",
+            ]
```

Comment:
This is a static array of elements.  Can this array be moved out into the enclosing scope?